src/renderer.ts

```diff
@@ -79,22 +78,19 @@ const getExtraSentryBreadcrumbs = ( targetElement: HTMLElement ) => {
 	return '';
 };
 
-Sentry.init(
-	{
-		debug: true,
-		beforeBreadcrumb( breadcrumb, hint ) {
-			const targetElement = hint?.event?.target;
+Sentry.init( {
+	debug: true,
+	beforeBreadcrumb( breadcrumb, hint ) {
+		const targetElement = hint?.event?.target;
 
-			if ( breadcrumb.category === 'ui.click' && targetElement ) {
-				breadcrumb.message =
-					( breadcrumb.message || '' ) + getExtraSentryBreadcrumbs( targetElement );
-			}
+		if ( breadcrumb.category === 'ui.click' && targetElement ) {
+			breadcrumb.message =
+				( breadcrumb.message || '' ) + getExtraSentryBreadcrumbs( targetElement );
+		}
 
-			return breadcrumb;
-		},
+		return breadcrumb;
 	},
-	reactInit
```

Comment:
When I keep the `reactInit`, it breaks the renderer. Sentry docs say we can pass it if we use framework specific SDK: [Framework-Specific SDKs](https://docs.sentry.io/platforms/javascript/guides/electron/#framework-specific-sdks). 

@nightnei could you please advise how to test breadcrumb code with that change?

Reply:
I can't test it, sicne looks like we have and issue in Sentry with limitations:
<img width="1235" alt="Screenshot 2025-04-23 at 14 26 54" src="https://github.com/user-attachments/assets/4c609b16-732a-40c4-995b-415caa505557" />

Also more info here: p1745411730036399-slack-C04GESRBWKW